src/common/schemas/DBRP.yml

```diff
@@ -19,6 +19,9 @@
     default:
       type: boolean
       description: Mapping represents the default retention policy for the database specified.
+    virtual:
+      type: boolean
```

Comment:
Is this field added to the response in cloud as well or is this oss only?

Reply:
It's currently only OSS, but I think this auto-mapping DBRP feature is probably something that would be ported to cloud as well.